.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+cache: pip
+python:
+    - "2.7"
+    - "3.4"
+    - "3.5"
+    - "3.5-dev"
+    - "3.6"
+    - "3.6-dev"
+    - "3.7-dev"
+matrix:
+    allow_failures:
+        - python: "2.7"
+        - python: "3.4"
+        - python: "3.7-dev"
+addons:
+    apt:
+        packages:
+            - dssp
+env:
+    - VERMOUTH_TEST_DSSP=mkdssp
+install:
+    - pip install pylint
+    - pip install --upgrade .
+script:
+    - pytest -vv vermouth
```

Comment:
-vv? Is that nescessary?

Reply:
The main point is to display the whole difference when an assert fails. My experience is that you will likely rerun the failing tests with -vv so having it from the start saves some time.